### DISCUSSION TITLE
July 2023 Melbourne MicroPython Meetup

### DISCUSSION THREAD

**Discussion by mattytrentini** [micropython/micropython#12080] [General]
The [July Melbourne MicroPython Meetup](https://www.meetup.com/micropython-meetup/events/294346484/) is on Wednesday!

It's in the evening in Melbourne, but you can see the [time in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=July+Melbourne+MicroPython+Meetup&iso=20230726T1830&p1=152&ah=3). It'll be held in-person but will be [live-streamed on zoom](https://planetinnovation.zoom.us/j/99861130556?pwd=RnV3YXU5WU5DYWh2WVllR3dMU0Ixdz09) and recorded.

Sean will be giving a News Roundup and @dpgeorge will be discussing some of the recently merged features and give us the low-down on the upcoming v1.21. We'll post slides and videos when they're available.

Hope to see you there, or on the stream!

**Comment by robert-hh** [comment]
Today it's Monday 24. Until Wednesday it will be a long night!

**Reply by mattytrentini** [reply]
We're really keen on MicroPython here in Melbourne! 😜 

Apologies, a copy 'n' paste error, now fixed. Thanks for pointing it out!

**Reply by scruss** [reply]
Ah, the luxury of a 5:15 am start for me! That's almost the normal time I get up.

**Comment by bidrohini** [comment]
Hope it will be a successful meet-up.

**Reply by ma261065** [reply]
Is this a monthly occurrence? If so, are the details for the August meetup finalised?

**Reply by mattytrentini** [reply]
The [August Meetup](https://www.meetup.com/micropython-meetup/events/295213484/) is locked in, though I haven't updated the event description yet. 

The loose plan - and it may yet change! - is that [Simon Pickett](https://twitter.com/simon_prickett/) is hoping to pull together a Redis/MicroPython demo, presenting remotely from the UK. We'll have also put [PyCon AU](https://2023.pycon.org.au/) behind us where @dpgeorge, @jimmo and I will have given talks - we'll give a brief summary of the event.

**Comment by robert-hh** [comment]
> Is this a monthly occurrence?

Yes. Every 4th Wednesday of a month at 6:30 PM Melbourne local time.

**Reply by Josverl** [reply]
thanks for the predictability, need to plan for an early lunchbreak on these days 🍕

**Comment by mattytrentini** [comment]
We use [Meetup](https://www.meetup.com/MicroPython-Meetup/) to organise the events. You can register for notifications as the event draws near!

**Comment by mattytrentini** [comment]
Also, the [July News Roundup slides](https://docs.google.com/presentation/d/e/2PACX-1vRnVrGNqTYGe9u3rMdRIn-opOmep3ZjBkL0w8DAG6rY9zI4f92jhnNDpjuh9VHyciBhcff0q6qAXA30/pub) are available. Big thanks to Sean for taking care of them this month!

Videos will be available soon.

**Reply by mattytrentini** [reply]
Videos are now available!

[Sean delivers the monthly News Roundup](https://www.youtube.com/watch?v=MQ3U6iJqgZE) and [Damien provides an overview of the recently-merged features](https://www.youtube.com/watch?v=QWFN-ewJtCE).

Enjoy!
